### DISCUSSION TITLE
Where does Unsaved Original Data Streams go?

### DISCUSSION THREAD

**Discussion by svk27** [openobserve/openobserve#5079] [Q&A]
Hey everyone, I'm just trying out open observe, and it is definitely something!
What I don't understand is., where does the log/stream data get's stored at when the "Store Original Data" is set to off.

For example, the below streams has a similar number of rows, except, one has "Store Original Data" set to "on".
![Screenshot 2024-11-16 214912](https://github.com/user-attachments/assets/2461fa4d-fff8-4695-bc74-8293b04e3c9a)

**For more context:**
I'm using Easypanel to host and manage the Docker container, with no volume/storage mounts attached, 
using AWS S3 and postgres for HA.

So basically, if I were to redeploy the container, it should purge everything, but it is not doing that, so I am not sure.

Sorry in advance if I am not able to see something obvious here, I've checked docs as much as I could, and found nothing specific to this!
Thanks!

**Comment by prabhatsharma** [comment]
No size/records being shown seems to be a bug. Can you please create a github issue.

If you are using s3 and postgres it should not purge everything if you are still pointing to same postgres and s3 since they have the data already.

**Reply by svk27** [reply]
Surely, I can do that. 
Let me clarify again as well., 
Organizations being the same suppose "abc", under which I had 2 streams "default" and "new".
the default one has "Store Original Data" set to "off".
the new one has "Store Original Data" set to "on".

It should not purge anything as s3 and postgres is there, true but, where is the data of "default" getting stored, as I don't see it in s3., not sure if it gets stored in postgres., even if it stores in my docker vps., after a redeply or a restart it should just go away, i tried doing it all testing it all out, but the data was there.

Let me create a github issue out of this info!
